### DISCUSSION TITLE
Processor停止执行任务了，感谢帮忙解答

### DISCUSSION THREAD

**Discussion by Logan80s** [midwayjs/midway#3590] [Q&A]
### Describe the problem（描述问题）

从日志里面发现定时任务在2天前就不再打印日志，判断已经停止执行任务了，应用日志没有发现什么error的信息，重启应用服务也没有作用，请问怎么让任务重新跑起来，发生这种情况的原因可能是什么
代码如下：

```
import { IProcessor, Processor } from '@midwayjs/bull';
import { FORMAT } from '@midwayjs/core';
import { Inject } from '@midwayjs/decorator';
import { ChannelService } from '../modules/channel/channel.service';

@Processor('evaluationCountTask', {
  repeat: {
    cron: FORMAT.CRONTAB.EVERY_PER_5_SECOND,
  },
})
export class EvaluationCountTask implements IProcessor {
  @Inject() private channelService: ChannelService;

  async execute() {
    await this.channelService.exeCountTask();
  }
}
```
redis中的情况如下：
![1](https://github.com/midwayjs/midway/assets/28385352/46a47236-f9ed-43dd-8dfa-38574f1f7b8f)

### Midway Versions（Midway 版本）

✓ @midwayjs/faas-typings(not installed)
✓ @midwayjs/fc-starter(not installed)
✓ @midwayjs/serverless-http-parser(not installed)
✓ @midwayjs/async-hooks-context-manager(3.12.3)
✓ @midwayjs/axios(not installed)
✓ @midwayjs/bootstrap(3.12.5)
✓ @midwayjs/bull(3.12.10)
✓ @midwayjs/bull-board(3.12.10)
✓ @midwayjs/cache(3.12.10)
✓ @midwayjs/captcha(3.12.10)
✓ @midwayjs/casbin(not installed)
✓ @midwayjs/casbin-redis-adapter(not installed)
✓ @midwayjs/casbin-typeorm-adapter(not installed)
✓ @midwayjs/code-dye(not installed)
✓ @midwayjs/consul(not installed)
✓ @midwayjs/core(3.12.3)
✓ @midwayjs/cos(not installed)
✓ @midwayjs/cron(not installed)
✓ @midwayjs/cross-domain(3.12.10)
✓ @midwayjs/decorator(3.12.3)
✓ @midwayjs/etcd(not installed)
✓ @midwayjs/express-session(not installed)
✓ @midwayjs/faas(not installed)
✓ @midwayjs/grpc(not installed)
✓ @midwayjs/http-proxy(not installed)
✓ @midwayjs/i18n(3.12.10)
✓ @midwayjs/info(3.12.10)
✓ @midwayjs/jwt(3.12.10)
✓ @midwayjs/kafka(not installed)
✓ @midwayjs/leoric(not installed)
✓ @midwayjs/mikro(not installed)
✓ @midwayjs/mock(3.12.10)
✓ @midwayjs/mongoose(3.12.10)
✓ @midwayjs/oss(3.12.10)
✓ @midwayjs/otel(not installed)
✓ @midwayjs/passport(not installed)
✓ @midwayjs/process-agent(not installed)
✓ @midwayjs/prometheus(not installed)
✓ @midwayjs/prometheus-socket-io(not installed)
✓ @midwayjs/rabbitmq(not installed)
✓ @midwayjs/redis(3.12.10)
✓ @midwayjs/security(not installed)
✓ @midwayjs/sequelize(not installed)
✓ @midwayjs/session(3.12.10)
✓ @midwayjs/socketio(not installed)
✓ @midwayjs/static-file(not installed)
✓ @midwayjs/swagger(3.12.10)
✓ @midwayjs/tablestore(not installed)
✓ @midwayjs/tags(not installed)
✓ @midwayjs/typegoose(3.12.10)
✓ @midwayjs/typeorm(not installed)
✓ @midwayjs/upload(3.12.10)
✓ @midwayjs/validate(3.12.10)
✓ @midwayjs/version(3.12.10)
✓ @midwayjs/view(not installed)
✓ @midwayjs/view-ejs(not installed)
✓ @midwayjs/view-nunjucks(not installed)
✓ @midwayjs/web(not installed)
✓ @midwayjs/express(not installed)
✓ @midwayjs/koa(3.12.10)
✓ @midwayjs/ws(not installed)

**Comment by czy88840616** [comment]
missing key ?看起来是 bull 中的任务被删除了，更细节的内容，可能要去 bull issue 中去寻找了。
